app/src/components/items/viewer/fallback.tsx

```diff
@@ -18,8 +19,8 @@ interface ItemProps {
 const ItemMediaViewerFallback = ({ item }: ItemProps) => {
   // hacky way to generate the link
   // this assumes that the first Library Location in the array is the Division link
-  // TODO: map DCFL Division link to nypl.org Division link
-  const divisionLink = item?.metadata?.locations?.split("<br>")[0];
+  // const divisionLink =
+  //   item.divisionLink ? item.divisionLink : item?.metadata?.locations?.split("<br>")[0];
```

Comment:
can we just remove the commented out code?

Reply:
yup